src/itemdrop.zig

```diff
@@ -700,10 +700,7 @@ pub const ItemDropRenderer = struct { // MARK: ItemDropRenderer
 	}
 
 	fn bindLightUniform(light: u32, ambientLight: Vec3f) void {
-		c.glUniform3fv(itemUniforms.ambientLight, 1, @ptrCast(&@max(
-			ambientLight*@as(Vec3f, @splat(@as(f32, @floatFromInt(light >> 24))/255)),
-			Vec3f{@floatFromInt(light >> 16 & 255), @floatFromInt(light >> 8 & 255), @floatFromInt(light & 255)}/@as(Vec3f, @splat(255))
-		)));
+		c.glUniform3fv(itemUniforms.ambientLight, 1, @ptrCast(&@max(ambientLight*@as(Vec3f, @splat(@as(f32, @floatFromInt(light >> 24))/255)), Vec3f{@floatFromInt(light >> 16 & 255), @floatFromInt(light >> 8 & 255), @floatFromInt(light & 255)}/@as(Vec3f, @splat(255)))));
```

Comment:
If you want to keep it in separate lines, you can add a trailing comma after the last argument, as a hint for the formatter.